core/API/DataTablePostProcessor.php

```diff
@@ -261,11 +261,21 @@ public function applyProcessedMetricsGenericFilters($dataTable)
             // ignore
         }
 
+        $goalsToProcess = null;
+        try {
+            $goalsToProcess = Common::getRequestVar('filter_show_goal_columns_process_goals', null, 'string', $this->request);
```

Comment:
is `filter_show_goal_columns_process_goals`  a new parameter? is it possible to add some test for this feature?

Reply:
It's now used by the goals visualization so it should already be covered. It's not really an exposed API, it's just for ProxySite.